Option to change layout extension

gijskooij opened the pull request:
:D

mohitsinghs commented:
Looks Good Now. Could you please update readme as well.

mohitsinghs commented:
Thank you for the pull request. Builds were failing on nodejs 4 due to reason unrelated to your changes. I removed nodejs 4 and added nodejs 10 and now everything is green. 🎉

gijskooij commented:
Heya Mohit,

I saw your change to the readme.

Have a great day!
Gijs.
ᐧ

On Mon, Apr 30, 2018 at 7:39 PM, Mohit Singh <notifications@github.com>
wrote:

> Thank you for the pull request. Builds were failing on nodejs 4 due to
> reason unrelated to your changes. I removed nodejs 4 and added nodejs 10
> and now everything is green. 🎉
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mohitsinghs/gulp-nunjucks-md/pull/13#issuecomment-385473086>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF58A3QjV9Nz_vmMCNTJ3LxPyaJo6BhBks5tt0xogaJpZM4Ts6MA>
> .
>



-- 

_________________________________

Gijs Kooij

*Greenmedia* <http://www.greenmedia.nl>
Coenecoop 750
2741 PW Waddinxveen

Bel me op: 06-18223475
*https://www.linkedin.com/in/gkooij* <https://www.linkedin.com/in/gkooij>

*Good actions give strength to ourselves *
*and inspire good actions in others.*
